controllers/cloud.redhat.com/clowdjobinvocation_controller.go

```diff
@@ -135,7 +135,7 @@ func (r *ClowdJobInvocationReconciler) Reconcile(req ctrl.Request) (ctrl.Result,
 		// Match the crd.Job name to the JobTemplate in ClowdApp
 		job, err := getJobFromName(jobName, &app)
 		if err != nil {
-			r.Recorder.Eventf(&app, "Error", "JobNameMissing", "ClowdApp [%s] has no job named", cji.Spec.AppName, jobName)
+			r.Recorder.Eventf(&app, "Warning", "JobNameMissing", "ClowdApp [%s] has no job named", cji.Spec.AppName, jobName)
```

Comment:
I just realised - I think you are passing more arguments here than your format string knows how to decode.

Reply:
.... you are correct.